frontend/src/pages/CourseSelector/helper.js

```diff
@@ -0,0 +1,20 @@
+export const prepareUserPayload = (degree, planner) => {
+  const { startYear, courses } = planner;
+  const { programCode, specialisation, minor } = degree;
+
+  const specialisations = {};
+  specialisations[specialisation] = 1;
+  if (minor !== "") specialisations[minor] = 1;
+
+  let selectedCourses = {};
+  for (const course of courses.keys()) {
+    selectedCourses[course] = 70;
```

Comment:
is this some dummy value? I dont see the segnificance of the number `70` here.

Reply:
I think this was sano's changes he made to my branch to make things easier on the frontend side. Unsure